networks.json

```diff
@@ -1,8 +1,8 @@
 {
   "optimism": {
     "PositionRegistry": {
-      "address": "0x447BEbdE07F6115b2714fd3a28e311b9d4a9E6f6",
-      "startBlock": 128171866
+      "address": "0xeE156D8ea7b96a5524CcC3CF9283ab85E80E9534",
```

Comment:
And about the other networks?

Reply:
There is a way to create network templates to manage multiple chains (refs: https://thegraph.com/docs/en/subgraphs/developing/deploying/multiple-networks/#using-subgraphyaml-template)

Reply:
Multichain setup is the plan, it will be added in near future

Reply:
Apologies, I just realize that I did started the setup but never completed. I should have either removed networks.json or completed setup end to end.

Reply:
Done